src/transaction.js

```diff
@@ -245,9 +246,9 @@ export default class Transaction {
             input.fulfillment = fulfillmentUri
         })
 
-        const serializedTransaction =
+        const serializedSecondTransaction =
```

Comment:
This is not a proper way of calling it as the transaction is the same. You could use `serializedSignedTransaction` instead or any other that represents better that constant

Reply:
Yes, I was thinking it was a bade naming just want to check